website/docs/cloud-docs/api-docs/run.mdx

```diff
@@ -53,55 +53,56 @@ Alternatively, you can create a run with a pre-existing configuration version, e
+| `post_plan_completed`  | The post-plan phase of the run has completed.                                                                                                                                                                                                                                                                                                                                                                                               |
+| `planned_and_finished` | The completion of a run containing a plan only, or a run that produces a plan with no changes to apply. This is a final state.                                                                                                                                                                                                                                                                                                              |
+| `apply_queued`         | Once the changes in the plan have been confirmed, the run will transition to `apply_queued`. This status indicates that the run should start as soon as the backend services that run terraform have available capacity. In Terraform Cloud, you should seldom see this status, as our aim is to always have capacity. However, in Terraform Enterprise this status will be more common due to the self-hosted nature.                      |
+| `applying`             | The applying phase of a run is in progress.                                                                                                                                                                                                                                                                                                                                                                                                 |
+| `applied`              | The applying phase of a run has completed.                                                                                                                                                                                                                                                                                                                                                                                                  |
+| `discarded`            | The run has been discarded. This is a final state.                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
okay this is fair - thank you for explaining that to me. In this one case then I will allow passive voice since we know know who or what is doing the discarding!